association_backend/main.py

```diff
@@ -22,6 +23,9 @@
         Route("/stripe/setup", stripe_views.StripeSetupView),
         Route("/stripe/customer-portal", stripe_views.CustomerPortalView),
     ],
+    middleware=[
+        pastaporto_auth_middleware(PASTAPORTO_SECRET),
+    ],
```

Comment:
you only need this for the pastaporto, you will have in `context.auth.pastaporto` or `context.user`

Reply:
ok done